src/components/standalone/firewall/conntrack/ConntrackContent.vue

```diff
@@ -0,0 +1,184 @@
+<!--
+  Copyright (C) 2024 Nethesis S.r.l.
+  SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { useI18n } from 'vue-i18n'
+import { ubusCall } from '@/lib/standalone/ubus'
+import ConntrackRecordsTable from '@/components/standalone/firewall/conntrack/ConntrackRecordsTable.vue'
+import DeleteConntrackRecordModal from '@/components/standalone/firewall/conntrack/DeleteConntrackRecordModal.vue'
+import {
+  NeHeading,
+  NeInlineNotification,
+  NeButton,
+  NeSkeleton,
+  NeEmptyState,
+  NeTextInput
+} from '@nethesis/vue-components'
+import { getAxiosErrorMessage } from '@nethesis/vue-components'
+import { onMounted, ref, computed } from 'vue'
+
+const { t } = useI18n()
+
+export type ConntrackRecord = {
+  destination: string
+  destination_port?: string
+  destination_stats: {
+    bytes: string
+    packets: string
+  }
+  id: string
+  protocol: string
+  source: string
+  source_port?: string
+  source_stats: {
+    bytes: string
+    packets: string
+  }
+  state?: string
+  timeout: string
+}
+
+const isLoading = ref(false)
+const conntrackRecords = ref<ConntrackRecord[]>([])
+const error = ref({
+  notificationDescription: '',
+  notificationDetails: '',
+  notificationTitle: ''
+})
+const filter = ref('')
+const showDeleteModal = ref(false)
+
+onMounted(() => {
+  fetchConntrack()
+})
+
+const selectedItem = ref<ConntrackRecord>()
+
+async function fetchConntrack() {
+  error.value.notificationDescription = ''
+  error.value.notificationDetails = ''
+  closeDeleteModal()
```

Comment:
I understand this statement has been put here because of `record-deleted` event, but it looks a bit odd here. How about creating a function `onRecordDeleted` where we close the modal and then call `fetchConntrack()`?